src/components/gas/GasSpeedButton.js

```diff
@@ -173,9 +174,10 @@ const GasSpeedButton = ({
 
   const formatGasPrice = useCallback(
     animatedValue => {
-      if (animatedValue === null || isNaN(animatedValue)) {
+      if (gasIsNotReady || animatedValue === null || isNaN(animatedValue)) {
         return 0;
       }
+      setGasPriceReady(true);
```

Comment:
hmmm doesn't this mean this will be triggered on every interpolation between numbers